src/main/resources/tags/climbing_pick_strong_surfaces.json

```
@@ -11,6 +11,26 @@
 		{
 			"id" : "minecraft:basalt",
 			"required" : false
-		}
+		},
+		{
+			"id" : "minecraft:calcite",
+			"required" : false	
```

slight nitpick, remove trailing tabs
```suggestion
			"required" : false
```